lib/mutant.rb

```diff
@@ -200,7 +200,14 @@ class Config
       fail_fast:         false,
       includes:          EMPTY_ARRAY,
       integration:       Integration::Null,
-      isolation:         Mutant::Isolation::Fork,
+      isolation:         Mutant::Isolation::Fork.new(
+        devnull: ->(&block) { File.open(File::NULL, File::WRONLY, &block) },
```

Comment:
AFAIK you do not need `WRONLY` when you use `APPEND`. But I've to test this.

I personally prefer not to extract a constant in initialization code, because it stretches the intention about multiple lines.